abstract class Failure {
  final Exception? exception;

  const Failure({
    this.exception,
  });
}